source/geometries/Next100FieldCage.cc

```diff
@@ -255,8 +255,8 @@ void Next100FieldCage::DefineMaterials()
   copper_ = G4NistManager::Instance()->FindOrBuildMaterial("G4_Cu");
 
   /// Teflon for the light tube
-  teflon_ =
-  G4NistManager::Instance()->FindOrBuildMaterial("G4_TEFLON");
+  teflon_ = G4NistManager::Instance()->FindOrBuildMaterial("G4_TEFLON");
+  teflon_->SetMaterialPropertiesTable(new G4MaterialPropertiesTable());
```

Comment:
Perhaps you should add here a comment explaining why we are adding an empty `G4MaterialPropertiesTable`. It may be helpful for future reference.

Reply:
Done